{
  "comments": [
    {
      "key": {
        "uuid": "9433cdd6_4a1933c9",
        "filename": "gerrit-acceptance-tests/src/test/java/com/google/gerrit/acceptance/server/change/SubmittedTogetherIT.java",
        "patchSetId": 2
      },
      "lineNbr": 144,
      "author": {
        "id": 1011123
      },
      "writtenOn": "2016-06-16T01:47:00Z",
      "side": 1,
      "message": "Isn\u0027t it possible to use exception.expect(..) and exception.expectMessage(...) here?",
      "range": {
        "startLine": 137,
        "startChar": 6,
        "endLine": 144,
        "endChar": 7
      },
      "revId": "2fe91df9854bbd6502e99e96093d36e12e2152b0",
      "serverId": "173816e5-2b9a-37c3-8a2e-48639d4f1153",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "542dd575_68e8c3af",
        "filename": "gerrit-server/src/main/java/com/google/gerrit/server/change/SubmittedTogether.java",
        "patchSetId": 2
      },
      "lineNbr": 108,
      "author": {
        "id": 1011123
      },
      "writtenOn": "2016-06-16T01:47:00Z",
      "side": 1,
      "message": "\"that you cannot see\"?",
      "range": {
        "startLine": 108,
        "startChar": 51,
        "endLine": 108,
        "endChar": 65
      },
      "revId": "2fe91df9854bbd6502e99e96093d36e12e2152b0",
      "serverId": "173816e5-2b9a-37c3-8a2e-48639d4f1153",
      "unresolved": false
    }
  ]
}